lib/split/trial.rb

```diff
@@ -35,22 +54,25 @@ def complete!
     def choose!
       choose
       record!
+
+      @alternatives
     end
 
     def record!
```

Comment:
the good news is that it's only 2 redis commands regardless of how many users now. even deterministic, we still need to know whether the user is participating though. even if you moved it "offline" would it be like a background job? in that case there's actually probably more redis commands that happen just to create a resque job for it.